modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,96 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling mistakes.
+:keywords: autocorrect, capitalization, spell, spelling, spellcheck
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tierthree
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows {productname} to autocorrect a specified set of spelling errors or capitalization errors, as they are being typed.
+
+[[limitations]]
+== Limitations
+
+The {pluginname} plugin currently has the following limitations:
+
+. English is the only supported language.
+
+. Capitalization and autocorrect can both be turned off, but specific autocorrections cannot.
++
+For example, Spelling Autocorrect can be entirely turned off, but there is, currently, no way of instructing the plugin to ignore a particular string that it considers a mis-spelling. If *cna* (which the plugin treats as a mis-typing of *can*) is a desired string in a particular use-case, turning Spelling Autocorrect off is the currently recommended setting.
+
+. End-users cannot turn Autocorrect on or off on hand-held devices.
+
+[[known-issues]]
+== Known issues
+
+The {pluginname} plugin also has the following known issues:
+
+. Switching to a language other than English disables the Autocorrect and Capitalization menus
++
+When the `autocorrect` plugin loads, *Autocorrect* and *Capitalization* menu items are added to the *Tools* menu.
++
+If {productname} is then switched to a language other than English, both these menu items are disabled. And they are not automatically re-enabled if a user switches back to English.
+
+. Spelling Autocorrect is case-sensitive.
++
+As a consequence, mis-spelled words are not auto-corrected if they do not exactly match the plugin’s built-in correction set.
++
+For example, Spelling Autocorrect auto-corrects *teh* to *the*, but does not auto-correct *Teh* to *The*.
+
+. Spelling Autocorrect over-rides auto-capitalization.
++
+As a consequence of both this and Spelling Autocorrect being case-sensitive, if
++
+  * `autocorrect` and `capitalization` are both active; and
+  * a word from the specified spelling error set is mis-typed at the beginning of a sentence;
++
+the spelling is corrected but the initial letter is not capitalized.
+
+. Auto-capitalization does not work on text immediately following a soft return in non-empty {productname} documents.
++
+If
++
+  * a {productname} document contains any data other than a soft return character (Unicode U+2028, commonly entered by typing *Shift+Return*); and
+  * the character immediately preceding an entered string is a soft return character
++
+Auto-capitalization will not operate on the entered string.
+
+. Auto-capitalization interferes with media embeds.
+
+. Auto-capitalization interferes with `linkchecker`.
```

Comment:
Should these be moved to the bottom of the page? I don't think the first thing people want to see is page of issues with our new plugin. Also, sounds like case-sensitivity should be a feature rather than a known issue as I doubt it needs to be "fixed". The conflict is still an issue though.

Reply:
> Should these be moved to the bottom of the page?

Yes.

> Also, sounds like case-sensitivity should be a feature rather than a known issue as I doubt it needs to be "fixed". The conflict is still an issue though.

To paraphrase my commit message, a bit of thinking about this lead me to *re-structure the whole document and make multiple edits and corrections*.

Reply:
@brian-forte I assume the move was in relation this, but this is not how things normally are done so doing this would be inconsistent

Reply:
At the very least the common/basic usage information should be at the top. As for limitations, as I said it'd be inconsistent so if you're moving one you really should be moving all. For example https://www.tiny.cloud/docs/tinymce/6/export/#client-side-pdf has the basic usage and limitations together at the top just below the example and before the options.

Reply:
> I assume the move was in relation this, but this is not how things normally are done so doing this would be inconsistent

Yep. My bad. Was too quick on the cut-and-paste-and-get-re-writing front. The **Options** and **Events** sections now round out the page.

Reply:
That looks better thanks 👍